modules/task.js

```diff
@@ -0,0 +1,31 @@
+// export const Addlist = () => {
+//   lists.push({
+//     description: document.getElementById("add").value,
+//     completed: false,
+//     index: 0,
+//   });
+
+// (lists.description = document.getElementById("add").value),
+// (lists.completed = false),
+// (lists.index = 0);
+
+//   const card = `
+//     <li class = "list"><div class="check-input"><input type="checkbox" id="${
+//       lists.index - 1
+//     }" name="${lists.index}" value="${lists.index}">
+//         <label for="${lists.index}" class="task-description">${
+//     lists.description
+//   }</label></div>
+//         <span class="dots">
+//         <span class="close-task">
+//           <i class="fa-solid fa-rectangle-xmark"></i>
+//         </span>
+//         <span class="edit-task">
+//         <i class="fa-solid fa-ellipsis-vertical"></i>
+//         </span>
+//       </span></li>`;
+//   document.getElementById("list-block").innerHTML += card;
+//   console.log(lists.length);
+
+//   //
+// };
```

Comment:
- [ ] Make sure you remove all commented code lines to make your code look professional and clean or you can even delete the file as it is not used anywhere.